src/components/thumbnail/Thumbnail.tsx

```diff
@@ -46,13 +48,15 @@ export const Thumbnail = ({
           </div>
         </div>
         <div className='thumbnail'>
-          {bookmarks.includes(movieData) ? (
+          {bookmarkedMovie ? (
             <i
+              data-testid='bookmarkIcon'
```

Comment:
Think its probably better for accessibility if we make this a button instead. Then we can match the test on the button rather than an ID